docs/UserGuide.md

```diff
@@ -63,8 +63,18 @@ Clears the current session of patientist and restores it to empty state.
 
 [Go back to [Table of Contents](#table-of-contents)]
 
-___
 
+---
+## Viewing the details of a specific person: view
+Lists the full detail of a specific person, including their name, patient ID and tags in the main window.
+
+**Format: `view INDEX`**
+
+**Examples:**
+**`view 1`** will display all the information associated with the 1st person shown on the GUI.
+
+[Go back to [Table of Contents](#table-of-contents)]
+___
```

Comment:
This should be `---` right.

Reply:
Wait is there a difference, I just copy pasted so this was the original line

Reply:
Oh then maybe it was a mistake from before already. It looks like this on your fork.

<img width="767" alt="Screenshot 2023-04-05 at 2 44 37 PM" src="https://user-images.githubusercontent.com/86766803/230002017-46a4e6a1-03ef-4608-8507-b37308e9269d.png">

Reply:
Oh oops, I’ll fix that tonight